src/components/aktivitetskrav/vurdering/AvventAktivitetskravSkjema.tsx

```diff
@@ -110,7 +110,11 @@ export const AvventAktivitetskravSkjema = ({
           <Button loading={vurderAktivitetskrav.isLoading} type="submit">
             {texts.lagre}
           </Button>
-          <Button variant="tertiary" onClick={() => setModalOpen(false)}>
+          <Button
+            type="button"
```

Comment:
Er `type`-feltet required nå? Føles kanskje litt overflødig å type en `Button`-komponent med `button` 🥲

Reply:
Hmja, måtte i hvert fall gjøre det sånn for at ikke form skulle bli submitted her :disappointed:

Reply:
Ahaa skjønner 👍🏼